docs/log-correlation.asciidoc

```diff
@@ -0,0 +1,172 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/java[elastic.co]
+endif::[]
+
+[[log-correlation]]
+== Log correlation
+
+
+The Elastic APM .NET agent provides integrations for popular logging frameworks, which take care of
+injecting trace ID fields into your application's log records. Currently supported logging frameworks are:
+
+- Serilog
+- NLog
+
+If your favorite logging framework is not already supported, there are two other options:
+
+* Open a feature request, or contribute code, for additional support, as described in https://github.com/elastic/apm-agent-dotnet/blob/master/CONTRIBUTING.md[CONTRIBUTING.md].
+* Manually inject trace IDs into log records, as described below in <<log-correlation-manual>>.
+
+Regardless of how you integrate APM with logging, you can use {filebeat-ref}[Filebeat] to
+send your logs to Elasticsearch, in order to correlate your traces and logs and link from
+APM to the {apm-app-ref}/xpack-logs.html[Logs app].
+
+[[serilog]]
+== Serilog
+
+We offer a https://github.com/serilog/serilog/wiki/Enrichment[Serilog Enricher] that adds the trace id to every log line that is created during an active trace.
```

Comment:
I'm not sure if this external link is needed. I just don't want to assume that everyone knows what a serilog enricher is - on the other hand, serilog users probably will know it. 

Maybe it's useful if someone is just researching logging frameworks and wants to know what we support. 

But if the external link is a no-go, I'm ok to remove. 

Same applies to `LayoutRenderers` for NLog below.

Reply:
I think the links make sense. Let's keep them.